copyright

```diff
@@ -1,4 +1,4 @@
-Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+﻿Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
```

Comment:
why is this a change? is there an invisible whitespace somewhere?

Reply:
no actual change, likely just something that was added & deleted or deleted & re-added, which triggers the code

Reply:
![image](https://user-images.githubusercontent.com/20871346/46918217-15f11d00-cf95-11e8-85cc-2748b5596731.png)
`<U+FEFF>` is a [Byte Order Mark](https://en.wikipedia.org/wiki/Byte_order_mark). My guess is an someone's editor chomped this file between d6a6702 and 4457e33 in one of the unrelated pulls.